website/Makefile

```diff
@@ -76,6 +76,8 @@ run-production-site-locally:
 	python -m http.server 8000 --directory ./public --bind 127.0.0.1
 
 # Full local builds without Algolia updates (for debugging, link checking, etc.)
+quick-build: clean setup structured-data production-build
```

Comment:
Is the comment on this still accurate? I see it used to be a comment for `local-production-build`.